src/data.d.ts

```diff
@@ -99,11 +99,22 @@ declare global {
       Subject: string,
       From: string,
       To: string,
+      Cc: string,
       HTMLBody: string,
       ReplyTo: string,
       MessageStream: 'outbound'
     }
 
+    type Interview = {
+      date: string,
```

Comment:
using Date is a little messy bc it converts it to a Timestamp automatically, but I did make the swap

Reply:
actually do what u think is best for the type, but we should be storing in firebase the date object rather than a string

Reply:
cause i know that the Input component requires string, so it may be necessary for there to be string here, just make sure that firebase is not storing a string

Reply:
I don't think firebase lets you store a Date? so I'm storing it as a timestamp